File: QuickStarts/VB/EventBroker/Src/EventBrokerExtension/ISimpleEventBrokerConfiguration.vb
'===============================================================================
' Microsoft patterns & practices
' Unity Application Block
'===============================================================================
' Copyright © Microsoft Corporation.  All rights reserved.
' THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY
' OF ANY KIND, EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT
' LIMITED TO THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
' FITNESS FOR A PARTICULAR PURPOSE.
'===============================================================================

Imports Microsoft.Practices.Unity
Imports SimpleEventBroker

Namespace EventBrokerExtension
	Public Interface ISimpleEventBrokerConfiguration
		Inherits IUnityContainerExtensionConfigurator
		ReadOnly Property Broker() As EventBroker
	End Interface
End Namespace

File: quickstarts/CS/EventBroker/Src/EventBrokerExtension/ISimpleEventBrokerConfiguration.cs
// Copyright (c) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.

using Microsoft.Practices.Unity;
using SimpleEventBroker;

namespace EventBrokerExtension
{
    public interface ISimpleEventBrokerConfiguration : IUnityContainerExtensionConfigurator
    {
        EventBroker Broker { get; }
    }
}
